libs/proposals/src/components/proposals-list/use-column-defs.tsx

```diff
@@ -25,8 +26,40 @@ import type { ProposalListFieldsFragment } from '../../lib/proposals-data-provid
 import { VoteProgress } from '../voting-progress';
 import { ProposalActionsDropdown } from '../proposal-actions-dropdown';
 
-export const useColumnDefs = () => {
+export const MarketNameProposalCell = ({
+  value,
+  data,
+}: VegaICellRendererParams<
+  ProposalListFieldsFragment,
+  'terms.change.instrument.code'
+>) => {
   const { VEGA_TOKEN_URL } = useEnvironment();
+  const { change } = data?.terms || {};
+  if (change?.__typename === 'NewMarket' && VEGA_TOKEN_URL) {
+    const productType =
```

Comment:
This seems like a sensible helper function to live in the proposals lib (with a unit test)

Reply:
a helper has been added to the lib utils